resources/views/components/dashboard/edit-dashboard.blade.php

```diff
@@ -0,0 +1,54 @@
+@use('FluxErp\Enums\TimeFrameEnum')
+<div class="flex flex-col md:flex-row gap-1.5">
+    <div x-cloak x-show="!editGrid" class="flex flex-col md:flex-row gap-1.5 items-center text-sm">
+        <x-select
+            class="p-2"
+            :options="array_map(function (TimeFrameEnum $timeFrame) {
```

Comment:
just use TimeFrameEnum::valuesLocalized() and then set option-key-value (instead of option-value and option-label)